ipet/evaluation/IPETFilter.py

```diff
@@ -461,6 +506,12 @@ class IPETFilterGroup(IpetNode):
 
     editableAttributes = ["name", "filtertype"]
 
+    # global data frame and index that are reused for several filter groups
+    _glbdf_ = None
+    _glbindex_ = None
+    _glbinterrows_ = None
```

Comment:
This is the third one in question. I apologize for not being able to use the comment functionality of github.

Reply:
I added a comment for this attribute.